pkg/reconciler/ingress/ingress.go

```diff
@@ -66,6 +74,9 @@ const (
 var (
 	ingressResource  = v1alpha1.Resource("ingresses")
 	ingressFinalizer = ingressResource.String()
+
+	routeResource  = routev1.Resource("routes")
+	routeFinalizer = routeResource.String()
```

Comment:
I think we need to keep the name stable from k-o-i to ensure us being able to seamlessly replace this. I think the finalizer will need to be called "ocp-ingress" for now, see https://github.com/openshift-knative/serverless-operator/commit/45e698b0efaa194545931c3c5b65c50804d529ee#diff-c9ce312d55f6b0b252419615db832a83R296

Reply:
Yes, I agree! Thank you. Updated.